database/populatePG.js

```diff
@@ -0,0 +1,72 @@
+require('dotenv').config();
+const path = require('path');
+const { Pool } = require('pg');
+const moment = require('moment');
+
+const host = process.env.PG_HOST;
+const port = process.env.PG_PORT;
+const database = process.env.PG_DB;
+const username = process.env.PG_USER;
+const password = process.env.PG_PASS;
+
+const pool = new Pool({
+    host: host,
+    port: port,
+    database: database,
+    user: username,
+    password: password,
+});
+
+const authorCsv = path.join(`${__dirname}/data/authorData.csv`);
+
+const copyCSV = (pathToCopy, type) => {
+    const copyText = `COPY ${type}
+    FROM '${pathToCopy}'
+    DELIMITER ','
+    CSV HEADER;`;
+    return pool.query(copyText);
+};
+
+const seedDB = async () => Promise.resolve(pool.connect())
+    .then(() => pool.query('SELECT NOW()'))
+    .then((result) => {
+        console.log('DB connected at ', result.rows[0].now);
+        const createSql = `DROP TABLE IF EXISTS author;
```

Comment:
Is it possible to save the query in a separate .sql file and load the file inside of pool.query()? Will it be more organized? I saw you have a separate .sql file with a schema for films.

Reply:
The film schema was an example, it shouldn't have been included. I will remove that file.